### PR TITLE
Fix linter issues

### PR DESCRIPTION
**Pull request by mook-as**
This removes the majority of the linter overrides and fixes any resulting errors and warnings.

### CODE REVIEW

File: background.js [hunk 1]
```diff
@@ -257,27 +256,23 @@ async function linkResource(name, state) {
 }
 
 function handleFailure(payload) {
-  let errorCode, message, titlePart = null;
-  if (typeof (payload) == "number") {
+  let { errorCode, message, context: titlePart } = payload;
```
Comment (ericpromislow) [inline background.js:259]: Did eslint or Mark make this change?

Comment (mook-as) [reply background.js:259]: I did, in d2c994120ddbaf79289cd997ec8ded34348e3b87 (`ESLint: one-var: manual fixes`).

**Review by ericpromislow — APPROVED** [review]
All very straightforward.

The only thing I think we need is a `lint:fix` command with the `--fix` option to run during dev so we don't have to manually align spaces in structs and to mitigate the hand-slapping following using double-quotes on strings (especially all-caps strings).  CI should run without fixing and fail on any style violations.

**Comment by mook-as** [comment]
I'll do the `lint:fix` in a follow up, to keep the momentum up.

**Pull request merged**